modules/vaos/app/models/vaos/appointment_request_form.rb

```diff
@@ -82,17 +82,17 @@ def facility=(values_hash)
     # These values ought to be derived from MVI vs user input, except for inpatient and text_messaging_allowed
     def patient=(values_hash)
       @patient = {
-        display_name: "#{last_name}, #{first_name}",
-        first_name: first_name,
-        last_name: last_name,
+        display_name: "#{@user.last_name_mpi}, #{@user.first_name_mpi}",
```

Comment:
```suggestion
        display_name: [@user.last_name_mpi, @user.first_name_mpi].join(', '),
```

Since last name is nil checked this could be empty here resulting in `", John"`. Would the suggest code change work instead?

Reply:
Should mention that you'll need to call `compact` off the array to do what you're suggesting here